src/main/java/org/apache/mesos/scheduler/DefaultScheduler.java

```diff
@@ -279,7 +279,8 @@ public void run() {
                 // UNRESERVE and DESTROY (in the case of persistent volumes) Operations.
                 ResourceCleanerScheduler cleanerScheduler = getCleanerScheduler();
                 if (cleanerScheduler != null) {
-                    acceptedOffers.addAll(getCleanerScheduler().resourceOffers(driver, offers));
+                    unacceptedOffers = filterAcceptedOffers(unacceptedOffers, acceptedOffers);
```

Comment:
This fixes an error `W0926 08:37:13.355495 26116096 sched.cpp:1348] Attempting to accept an unknown offer 30b17bfc-685c-49a8-8fe5-e1c3fa6cf188-O7` when cleaner tries to unreserve resources and destroy volume.

This is what happens without the fix:
Scheduler receives an offer with both unreserved and reserved (remaining from the failed task) resources. Recovery scheduler accepts this offer to reserve resources and launch a new task instead of the failed one. It does not use reserved resources, because the task is marked as permanently failed. Next, resource cleaner detects unused reserved resources in the same offer, and accepts it with `UNRESERVE` and `DESTROY` operations, however it fails with the above message. Then cleaner successfully repeats operations on the next `resourceOffers`.